### ISSUE TITLE
Error: UNKNOWN_ERROR while getProducts on IOS

### ISSUE THREAD

**Issue by indapublic** [dooboolab/react-native-iap#1136]
### Version of react-native-iap

4.5.3

### Version of react-native

0.63.2

### Platforms you faced the error (IOS or Android or both?)

iOS

### Expected behavior

Products list should be received

### Actual behavior

Unknown error

### Tested environment (Emulator? Real Device?)

Real Device

### Steps to reproduce the behavior

```
initConnection().then((t) => {
	console.warn(t)	//	true
	clearProductsIOS()
	getProducts(["myproductid"])
		.then(products => {
			console.warn(products)
		})
		.catch(err => {
			console.error(err)	//	UNKNOWN ERROR
		})
})
```

**Comment by alexander-mozolevsky-epam** [comment]
Fresh news, this library probably won't work in production (got reject from Apple team), reject shows that the app can't connect to iTunes.

Probably it relates to 14.0.0 version, something has changed with that and we need to update this library.

**Comment by Kouznetsov** [comment]
ios 14 right ?

**Comment by indapublic** [comment]
@Kouznetsov Yes

**Comment by joeyscarim** [comment]
same issue as https://github.com/dooboolab/react-native-iap/issues/1118

**Comment by hyochan** [comment]
Closes via #1118
